src/screens/SendTokens.js

```diff
@@ -132,11 +128,11 @@ class SendTokens extends React.Component {
       }
       // some token was invalid, it will be on arg, which ones
       const tokenList = this.state.txTokens.filter(t => arg.data.includes(t.uid))
-      this.setState({
-        ledgerAlertModalTitle: t`Invalid custom tokens`,
-        ledgerAlertModalBody: this.renderAlertTokenList(tokenList),
-      });
-      $("#ledgerAlertModal").modal('show');
+      this.context.showModal(MODAL_TYPES.ALERT, {
+        id: 'ledgerAlertModal',
+        title: t`Invalid custom tokens`,
+        body: this.renderAlertTokenList(tokenList),
```

Comment:
There's. a `buttonName={t`Close`}` missing.

Reply:
Fixed in b66bd3ad0a3a763a971a36fe9fb177bb040ed65f